docs/CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
 ## Changelog for v2.1.0
 
-+ Fix javadoc issues.
\ No newline at end of file
++ Fix javadoc issues.
++ Fix geometry_columns view function in order to escape table name with reserved database word
```

Comment:
Thanks a lot
Please add the issue number 
Fix geometry_columns view function in order to escape table name with reserved database word (#1269)